sources/functions/utils

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+function dist_info() {
+	DIST_CODENAME="$(source /etc/os-release && echo "$VERSION_CODENAME")"
+	DIST_ID="$(source /etc/os-release && echo "$ID")"
```

Comment:
```suggestion
	OS_CODENAME="$(source /etc/os-release && echo "$VERSION_CODENAME")"
	OS_DISTRO="$(source /etc/os-release && echo "$ID")"
```

I think this would give more context and clarity over what these contain

Reply:
```suggestion
	DIST_CODENAME="$(grep VERSION_CODENAME /etc/os-release | cut -d= -f2)"
	DIST_ID="$(grep -E "^ID=" /etc/os-release | cut -d= -f2)"
```

If we want to make sure that we don't actually source the `ID` value. Not sure if it will or won't so a separate suggestion

Reply:
I think sourcing it in subshell has the same outcome as grepping it in this case but maybe a bit more predictable, but worth keeping in mind though.